src/wallet/wallet.ts

```diff
@@ -157,6 +189,66 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return this.authToken;
   }
 
+  /**
+   * Returns the balance for each token in tx, if the input/output belongs to this wallet
+   *
+   * This method is meant to keep compatibility with the old facade
+   *
+   * @param {Object} tx Transaction data with array of inputs and outputs
+   *
+   * @return {Object} Object with each token and it's balance in this tx for this wallet
+   **/
+  async getTxBalance(tx, optionsParam = {}) {
```

Comment:
Should we add this method to the `IHathorWallet` interface, since it's now a method that must be implemented?

Reply:
a16aa65f6bd739dfb8df7e9388de1644e3032661